src/Workspaces/Core/Portable/Rename/Renamer.cs

```diff
@@ -17,36 +17,27 @@ public static class Renamer
         public static Task<Solution> RenameSymbolAsync(
             Solution solution, ISymbol symbol, string newName, OptionSet optionSet, CancellationToken cancellationToken = default)
         {
-            return RenameSymbolAsync(
-                solution,
-                SymbolAndProjectId.Create(symbol, projectId: null),
```

Comment:
public entrypoint would now be albe to use OOP (which is coming in a followup PR).